src/main/java/seedu/address/logic/commands/TagFindCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.TagContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class TagFindCommand extends Command {
```

Comment:
I think `FindTagCommand`or `FindByTagCommand`  would be more apt/ sounds better.

Reply:
Thank you for the suggestion. Yes, I think that sounds clearer. I've changed it to `FindByTagCommand`. Thanks!